Makefile

```diff
@@ -41,7 +41,7 @@ all: clean lint test build
 
 .PHONY: lint
 lint:
-	$(GO_LINT) version || curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(GO_PATH)/bin
+	$(GO_LINT) version || curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/v1.42.1/install.sh | sh -s -- -b $(GO_PATH)/bin
```

Comment:
FYI @apache/skywalking-committers We noticed master and v1.42.1 are not consistent in format rules, which caused CI failing in lint checking.
We are locking the version now.

Reply:
Please use `-d <tag>` to appoint the version. So changed to `$(GO_LINT) version || curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(GO_PATH)/bin -d "v1.42.1"`.